### PR TITLE
Included that retrain_full = True does not include the user provided validation data in the docs. #1228

### LINKED ISSUE #1228: include that `retrain_full = True` does not include the user provided validation data in the docs.
It'd be worth including that `retrain_full = True` does not include the user provided validation data in the docs.

_Originally posted by @RossDeVito in https://github.com/microsoft/FLAML/issues/735#issuecomment-1735978213_

### PR DESCRIPTION
**Pull request by smty2018**
## Why are these changes needed?

Included that retrain_full = True does not include the user provided validation data in the docs. #1228

## Related issue number

Closes #1228 

## Checks

- [X] I've included any doc changes needed for https://microsoft.github.io/FLAML/. See https://microsoft.github.io/FLAML/docs/Contribute#documentation to build and test documentation locally.
- [X] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [X] I've made sure all auto checks have passed.

### CODE REVIEW

File: website/docs/Use-Cases/Task-Oriented-AutoML.md [hunk 1]
```diff
@@ -360,7 +360,8 @@ By default, flaml decides the resampling automatically according to the data siz
 For holdout, you can also set:
 * `split_ratio`: the fraction for validation data, 0.1 by default.
 * `X_val`, `y_val`: a separate validation dataset. When they are passed, the validation metrics will be computed against this given validation dataset. If they are not passed, then a validation dataset will be split from the training data and held out from training during the model search. After the model search, flaml will retrain the model with best configuration on the full training data.
-You can set`retrain_full` to be `False` to skip the final retraining or "budget" to ask flaml to do its best to retrain within the time budget.
+You can set`retrain_full` to be `False` to skip the final retraining or "budget" to ask flaml to do its best to retrain within the time budget.When `retrain_full` is set to `True`,the user-provided validation data is not used in the final retraining of the model.
```
Comment (sonichi) [inline website/docs/Use-Cases/Task-Oriented-AutoML.md:363]:
```suggestion
You can set`retrain_full` to be `False` to skip the final retraining or "budget" to ask flaml to do its best to retrain within the time budget. When `retrain_full` is set to `True`, the user-provided validation data is not used in the final retraining of the model.
```

**Review by sonichi — APPROVED** [review]
LGTM.

**Comment by dannycg1996** [comment]
Hi @thinkall, would it be possible to merge this PR? Was just looking at open PRs, and this one looks ready to merge.

File: website/docs/Use-Cases/Task-Oriented-AutoML.md [hunk 2]
```diff
@@ -391,9 +391,10 @@ By default, flaml decides the resampling automatically according to the data siz
...
-  You can set`retrain_full` to be `False` to skip the final retraining or "budget" to ask flaml to do its best to retrain within the time budget.
+* `split_ratio`: the fraction for validation data, 0.1 by default.
+* `X_val`, `y_val`: a separate validation dataset. When they are passed, the validation metrics will be computed against this given validation dataset. If they are not passed, then a validation dataset will be split from the training data and held out from training during the model search. After the model search, flaml will retrain the model with best configuration on the full training data.
+You can set`retrain_full` to be `False` to skip the final retraining or "budget" to ask flaml to do its best to retrain within the time budget. When `retrain_full` is set to `True`, the user-provided validation data is not used in the final retraining of the model.
+
```
Comment (thinkall) [inline website/docs/Use-Cases/Task-Oriented-AutoML.md:397]:
```suggestion
- `split_ratio`: the fraction for validation data, 0.1 by default.
- `X_val`, `y_val`: a separate validation dataset. When they are passed, the validation metrics will be computed against this given validation dataset. If they are not passed, then a validation dataset will be split from the training data and held out from training during the model search. After the model search, flaml will retrain the model with best configuration on the full training data.
You can set`retrain_full` to be `False` to skip the final retraining or "budget" to ask flaml to do its best to retrain within the time budget. When `retrain_full` is set to `True`, the user-provided validation data is not used in the final retraining of the model.

```
Just to keep consistency of the marker.

**Review by thinkall — APPROVED** [review]
LGTM! Thank you, @smty2018 and @dannycg1996!

File: website/docs/Use-Cases/Task-Oriented-AutoML.md [hunk 3]
```diff
@@ -393,7 +393,8 @@ For holdout, you can also set:
 
 - `split_ratio`: the fraction for validation data, 0.1 by default.
 - `X_val`, `y_val`: a separate validation dataset. When they are passed, the validation metrics will be computed against this given validation dataset. If they are not passed, then a validation dataset will be split from the training data and held out from training during the model search. After the model search, flaml will retrain the model with best configuration on the full training data.
-  You can set`retrain_full` to be `False` to skip the final retraining or "budget" to ask flaml to do its best to retrain within the time budget.
+You can set`retrain_full` to be `False` to skip the final retraining or "budget" to ask flaml to do its best to retrain within the time budget. When `retrain_full` is set to `True`, the user-provided validation data is not used in the final retraining of the model.
+
```
Comment (thinkall) [inline website/docs/Use-Cases/Task-Oriented-AutoML.md:397]:
```suggestion
  You can set`retrain_full` to be `False` to skip the final retraining or "budget" to ask flaml to do its best to retrain within the time budget. When `retrain_full` is set to `True`, the user-provided validation data is not used in the final retraining of the model.
```

**Pull request merged**